core-feature-pack/src/main/resources/content/docs/contrib/scripts/systemd/wildfly.service

```diff
@@ -5,12 +5,15 @@ Before=httpd.service
 
 [Service]
 Environment=LAUNCH_JBOSS_IN_BACKGROUND=1
+EnvironmentFile=-/etc/wildfly/wildfly.d/*
 EnvironmentFile=-/etc/wildfly/wildfly.conf
+Sockets=httpd.socket
```

Comment:
Well you can have systemd spawn them, which essentially solves this historical problem we are stuck with on non-custom kernels. I think this is what apache does as packaged in Fedora. See this discussion from last year in the kernel mailing list: https://lkml.org/lkml/2015/12/14/395